src/WebdriverClassicDriver.php

```diff
@@ -940,6 +940,9 @@ private function applyTimeouts(): void
                     case 'implicit':
                         $timeouts->implicitlyWait($param / 1000);
                         break;
+                    case 'page load':
+                    case 'pageLoad':
+                        // ☝️ For MinkSelenium2Driver backward compatibility
```

Comment:
Was an emoji added to the comment on purpose?

Reply:
Yeah... thought it would be some harmless fun.

Reply:
Although now that I think about it, perhaps it's more functionally useful to replace that comment with:
```php
@trigger_error(
    "Using `$type` timeout type is deprecated, please use `page` instead.",
    E_USER_DEPRECATED
);
```
What do you think?

Reply:
That makes sense.